samples/Azure.ResourceManager.Sample/Generated/Models/SampleUsage.cs

```diff
@@ -69,7 +68,8 @@ internal SampleUsage(UsageUnit unit, int currentValue, long limit, SampleUsageNa
         /// An enum describing the unit of usage measurement.
         /// Serialized Name: SampleUsage.unit
         /// </summary>
-        public UsageUnit Unit { get; }
+        public UsageUnit Unit { get; } = UsageUnit.Count;
```

Comment:
In this PR we also change the way we assign initial value for a required constant - in the same way as we do for required literal types in DPG